FLEDGE_implementation_overview.md

```diff
@@ -0,0 +1,64 @@
+
+For each bidder origin, Chrome determines a tentative priority - using each interest group’s fixed priority, or priority vector, if present.  Chrome then sorts the interest groups based on priority, removing any with priority vectors and negative priorities.  Interest groups with matching priorities are grouped by joining origin (to improve performance in case ExecutionMode is groupByOrigin).  Groups with matching joining origins and priority are randomly shuffled.  For each bidder origin, Chrome checks if any interest group participating in the auction has “enableBiddingSignalsPrioritization” set.  If not, Chrome applies the auction’s per-buyer size limit to the bidder’s interest groups.  Either way, the interest groups are now in a priority order that will affect the order in which other resources will be fetched, and worklets will be run in.  None of this affects additional bids.
+
+
+## Phase 2 Bidding and Scoring Phase
+
+### Starting Worklet Executors
+Once all interest groups are loaded, Chrome starts loading the seller worklet, and all bidder worklets participating in an auction.  To load a worklet, Chrome first checks if there is already a matching worklet “executor” (Same worklet origin, same javascript URL, wasm URL, trusted signals URL, same frame - frame match is required by the DevTools hooks.  It may be for a different auction / component auction, however).  If so, the executor is reused.  Note that different executors do not currently share any network fetches or Javascript contexts, although they do share the same global HTTP cache, so it’s generally a good idea to try and use the same URLs for all interest groups, to maximize executor reuse.  Also, interest groups that share an executor are guaranteed to be run in the priority order they were sorted into (this may change if trusted signals fetches are split up, though).  Interest groups that don’t share an executor are run on that basis of whichever has everything it needs to run first, runs first.
+
+Otherwise, Chrome requests a bidder or seller worklet process for the associated buyer/seller origin.  Buyers and sellers do not share processes, even if the origin is the same. There are separate limits on the number of buyer processes (10) and seller processes (3).  These numbers are not optimized, and subject to change.  On mobile, Chrome will generally reuse the single renderer process, instead of using more secure, isolated processes, due to resource constraints.  Within a process, all FLEDGE worklets use a single Javascript thread.  Once a process is assigned, Chrome creates an executor for the worklet, which starts loading the javascript and wasm URLs.  Because bidders share processes based on origin, this process means that all of a single bidder origin’s interest groups are assigned an executor (newly created or reused) at once.
```

Comment:
Scope is this auction, frame?

Reply:
These limits are global, for all of Chrome - unfortunately, processes are pretty expensive.  We should invest in seeing if we can up these numbers, though.  Clarified (Also clarified that it's one process per buyer/seller origin globally - including across frames)